src/components/SwapWidget/SwapWidget.tsx

```diff
@@ -192,6 +195,12 @@ const SwapWidget: FC = () => {
     baseTokenInfo?.address === nativeCurrencyAddress &&
     !!nativeCurrencySafeTransactionFee[baseTokenInfo.chainId];
 
+  useEffect(()=>{
+    if(showOrderSubmitted&&currOrder){
+      setOrderNonce((currOrder!.nonce!));
```

Comment:
this was the way i was able to get the nonce. i tried getting it from besTradeOption.order.nonce but that was not working well for me. is this a good way to get the nonce?